app/src/main/res/values/textStyles.xml

```diff
@@ -19,6 +19,18 @@
         <item name="android:textStyle">normal</item>
     </style>
 
+    <style name="TextAppearance.TitleLarge.Bold">
+        <item name="android:textSize">16sp</item>
+        <item name="android:textColor">@color/hint_text</item>
+        <item name="android:textStyle">bold</item>
+    </style>
+
+    <style name="TextAppearance.TitleLarge.Italic">
+        <item name="android:textSize">16sp</item>
+        <item name="android:textColor">@color/hint_text</item>
+        <item name="android:textStyle">italic</item>
+    </style>
+
```

Comment:
(Keeping in mind migration to Jetpack Compose) I'd prefer the following:

- use TitleMedium for the "Another person noted"
- use Body for the hint text and just set the italic + color in the layout itself
- remove these new text styles

Reply:
Resolved by fa183c21d